Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1178,7 +1178,7 @@ class BrowserViewController: UIViewController {
             //Disable any sort of privileged execution contexts
             //IE: The user must explicitly type OR must explicitly tap a bookmark they have saved.
```

Comment:
This comment must be updated.

Reply:
Comment updated, thanks for pointing this out.